packages/roosterjs-content-model-dom/lib/formatHandlers/utils/color.ts

```diff
@@ -96,6 +109,56 @@ export function setColor(
     isBackground: boolean,
     isDarkMode: boolean,
     darkColorHandler?: DarkColorHandler
+) {
+    const newColor = generateColor(element, color, isBackground, isDarkMode, darkColorHandler);
+
+    element.removeAttribute(isBackground ? 'bgcolor' : 'color');
+    element.style.setProperty(isBackground ? 'background-color' : 'color', newColor || null);
+}
+
+/**
+ * Set the given color to the table border
+ * @param element The element to set color to
+ * @param borderKey the style attribute of the border, ex: borderTop
+ * @param border the border values with the color in light mode
+ * @param isDarkMode Whether element is in dark mode now
+ * @param darkColorHandler @optional The dark color handler object to help manager dark mode color
+ */
+export function setColorOnBorder(
```

Comment:
Please refer to my previous comment, no need to have a separate `setColorOnBorder` function, but just set them together with border width and border style.